snippets/card-product.liquid

```diff
@@ -389,7 +389,7 @@
               data-min="{{ card_product.selected_or_first_available_variant.quantity_rule.min }}"
               id="quick-add-bulk-{{ card_product.selected_or_first_available_variant.id }}-{{ section.id }}"
               class="quick-add-bulk"
-              data-id="{{ card_product.selected_or_first_available_variant.id }}"
+              data-index="{{ card_product.selected_or_first_available_variant.id }}"
```

Comment:
I just changed this since it is what we were using in quick order list and cart too 🤔 but I agree it feels weird. 

I dont want to change it on this PR since it affects cart etc and increases scope of testing, but I will create a fast follow to ensure it's fixed

Reply:
ah no worries, sounds good 👍